thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/start-duration/template.hbs

```diff
@@ -1,7 +1,8 @@
 {{!-- TODO: Replace css classes below with te-anomaly-table styles --}}
-<p class="anomalies-table__start">
+{{#link-to 'rootcause' (query-params anomalyId=record.id)
+  class="te-anomaly-table__link"}}
   {{record.start}}
-</p>
+{{/link-to}}
 <p class="anomalies-table__duration">
   {{record.duration}}
```

Comment:
I think we can remove the `TODO` comment and the classname on line 6. I had it there, thinking that I was going to do something about it later, but it doesn't seem like it needs any additional styles to apply here.